.gitignore

```diff
@@ -35,3 +35,5 @@ Configs/user.xcconfig
 Configs/bundleIdentifier.xcconfig
 tmp/
 
+## Local Files
```

Comment:
What is this change about?

Reply:
I put my local design file for affinity designer into a local folder. I felt that these kind of file are not supposed to be in a git repo? So I put them into Local/ and gitignored it.

Reply:
Gotcha, I would recommend keeping those files in a different folder outside the project or checking them in under docs/resources if you think they are a good resource for others to have.

gitignoring a local folder seems fine as well, just higher chance you may lose those files since anything gitignored can easily be deleted. Only nit I have is to rename the folder `Local/` -> `local/`

Reply:
Okay! I'll put them in another folder. :-)

Reply:
👍 Can merge this in once Local is renamed to local or just removed from gitignore.